test/utils/fixture/fixture.go

```diff
@@ -95,10 +102,43 @@ func Teardown(ft *Fixture) error {
 		return nil
 	}
 
+	if initialPFP != nil {
+		ginkgo.By("wait for NRTs to settle")
+		equal, err := isPFPEqual(ft.Client, initialPFP)
+		if err != nil {
+			return err
+		}
+		if !equal {
+			return errors.New("NRT data is not settled to the initial, PFP is different")
+		}
+	}
+
 	Cooldown()
```

Comment:
not sure we still need this cooldown time. before removing it need to run the suite few times with 0 cooldown and see. note that cooldown is also used within the tests and not only here

Reply:
I think the logic you added should be put inside `Cooldown`.
If no initialPFP state available, just sleep like today. If available, don't sleep and wait for PFP to be restored (or fail)

Reply:
good point! thanks for raising it. will handle it.